resources/web/style/base_styles.pcss

```diff
@@ -30,4 +30,67 @@
   details {
     margin-bottom: 1.15em;
   }
+
+  .content-container {
+    width: 100%;
+    padding-right: 20px;
+    padding-left: 20px;
+    margin-right: auto;
+    margin-left: auto;
+    @media screen and (min-width: 1560px) {
+      max-width: 1500px;
+    }
+  }
```

Comment:
Is there an advantage to using this over Bootstrap's built-in `container-fluid` class? The main difference looks to be the media query—does it make more sense to use the Bootstrap class and just add the query to this style sheet?

https://github.com/elastic/docs/blob/4110e2b593906d7ed7c1e1c8c93c9f2d8317d8a1/resources/web/lib/bootstrap/bootstrap.css#L621-L627

Reply:
I didn't know about `container-fluid`! Feel free to make the switch. 🙂